rfcs/proposed/task_group_dynamic_dependencies/README.md

```diff
@@ -366,13 +340,26 @@ This task tree matches the one shown earlier for merge-sort.
 
 ## Open Questions in Design
 
-Some open questions that remain:
-
+Questions that should be addressed before moving into `experimental` and providing a preview feature:
 - Are the suggested APIs sufficient?
+- Are there additional use cases that should be considered that we missed in our analysis?
+- Semantics for `task_arena::enqueue` method that takes a `task_handle` object should be defined for all possible states of the handle.
+- What are the performance targets for this feature?
+- What are the exit criteria for a feature to become fully supported?
+- The API applicability for other usage scenarios should be checked, e.g.:
+  * A wavefront example representing tasks with multiple successors and predecessors
+  * Fibonacci example
+  * A two-stage parallel scan
+  * Other non-trivial divide-and-conquer patterns
```

Comment:
Do we want all these scenarios to be investigated before going experimental? Maybe some can be looked at later.

Reply:
Moved to the second list